src/include/catalog/index_schema.h

```diff
@@ -116,8 +161,9 @@ class IndexSchema {
     std::string name_;
     indexkeycol_oid_t oid_;
     uint32_t packed_type_;
-    common::ManagedPointer<const parser::AbstractExpression> definition_;
-    std::string serialized_expression_;
+
+    // TODO(John) this should go back to being a raw pointer once #386 is in
```

Comment:
Outdated TODO. Not sure if we know what PR to reference at this point.

Reply:
Changed to reference the new issue for smart pointers (#489).